src/creature.cpp

```diff
@@ -249,6 +249,11 @@ bool Creature::getNextStep(Direction& dir, uint32_t&)
 
 void Creature::startAutoWalk(const std::forward_list<Direction>& listDir)
 {
+	if (getPlayer() && getPlayer()->isNoMove()) {
```

Comment:
Rename `isNoMove()` to `canMove()` and use that throughout this patch (should default to true obviously).

Reply:
But I wanted to call `sendCancelWalk` only if player is freezed. So maybe it could be `canNoMove` instead of `isNoMove`?

Reply:
But I wanted to call `sendCancelWalk` only if player is freezed. So maybe it could be `canNoMove` instead of `isNoMove`?

Reply:
If `cannotMove` is already used in create.h, then it should be used for player as well.

Reply:
Nomenclature changed.